CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The types of changes are:
 - `Security` in case of vulnerabilities.
 
 ## [Unreleased](https://github.com/ethyca/fides/compare/2.39.0...main)
+- Adds last_monitored and enabled attributes to MonitorConfig [#4991](https://github.com/ethyca/fides/pull/4991)
```

Comment:
you'll want to put this under an `### Added` subsection:

```suggestion
## [Unreleased](https://github.com/ethyca/fides/compare/2.39.0...main)

### Added
- Adds last_monitored and enabled attributes to MonitorConfig [#4991](https://github.com/ethyca/fides/pull/4991)
```

Reply:
Fixed in [07452cf](https://github.com/ethyca/fides/pull/4991/commits/07452cf3815b44a9474036895075bbf28e4f8806)